AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/QtPyScriptCanvasNodeInspector.py

```diff
@@ -0,0 +1,111 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Object to house all the Qt Objects and behavior used in testing the script canvas Node Inspector.
+
+***Important Note: The QWidget hierarchy of the node inspector changes based on what node/variable is selected***
+"""
+
+from editor_python_test_tools.utils import TestHelper as helper
+from PySide2 import QtWidgets, QtTest, QtCore
+import pyside_utils
+from consts.scripting import (PROPERTY_EDITOR_QT, INITIAL_VALUE_SOURCE_QT, NODE_INSPECTOR_QT)
+from consts.general import (WAIT_TIME_SEC_3)
+from enum import IntEnum
+
+class CheckBoxStates(IntEnum):
+    Off = 0
+    On = 1
+
```

Comment:
Seems like great behavior for the `QtPyCommon ` class to hold onto.

Reply:
moved this to the common class and now associated classes import it